lib/lotus/view.rb

```diff
@@ -268,9 +268,9 @@ def self.included(base)
       conf.add_view(base)
 
       base.class_eval do
-        extend Inheritable.dup
-        extend Dsl.dup
-        extend Rendering.dup
+        extend Inheritable
+        extend Dsl
+        extend Rendering
         extend Escape.dup
```

Comment:
This guy doesn't need it because it doesn't have class variables, so no issue with scope ;)